src/Spryker/Zed/Queue/Business/Process/ProcessManager.php

```diff
@@ -103,7 +103,9 @@ protected function releaseIdleProcesses($processIds)
             }
         }
 
-        $this->deleteProcesses($cleanupProcesses);
+        if ($cleanupProcesses) {
```

Comment:
please use a more strict comparison
 ```suggestion
        if ($cleanupProcesses !== []) {
```